pkg/capacity/capacity.go

```diff
@@ -107,6 +115,28 @@ var (
 		Factor:   1.1,
 		Steps:    10,
 	}
+
+	objectsGoalDesc = metrics.NewDesc(
+		"csistoragecapacities_desired_goal",
+		"Number of CSIStorageCapacity objects that are supposed to be managed automatically.",
+		nil, nil,
+		metrics.ALPHA,
+		"",
+	)
+	objectsCurrentDesc = metrics.NewDesc(
+		"csistoragecapacities_desired_current",
+		"Number of CSIStorageCapacity objects that exist and are supposed to be managed automatically.",
+		nil, nil,
+		metrics.ALPHA,
+		"",
+	)
+	objectsObsoleteDesc = metrics.NewDesc(
+		"csistoragecapacities_obsolete",
+		"Number of CSIStorageCapacity objects that exist and will be deleted automatically. Objects that exist and may need an update are not considered obsolete and therefore not included in this value.",
```

Comment:
Your comment makes sense to me and I don't have a strong preference either way whether to add that part of the description

Reply:
@msau42 I'll leave it as-is unless you have some other preference.

Reply:
The comment was the one in https://github.com/kubernetes-csi/external-provisioner/pull/579#discussion_r599576956